nautobot/extras/templates/extras/role_retrieve.html

```diff
@@ -46,20 +46,23 @@
 {% endblock content_left_page %}
 
 {% block content_full_width_page %}
+    {% if "dcim | device" in content_types %}
```

Comment:
Could you add a test case asserting only expected tables are shown

Reply:
I've updated the view to provide those flags.  For the test, do you have any example code to assert some html is not in the body of a page?

Reply:
Actually, I think we can assert this via extra_context using the view

Reply:
`self.assertNotIn` should do the trick

Reply:
> Actually, I think we can assert this via extra_context using the view

Im not sure this would test if the table is shown in HTML

Reply:
Can you please point out an example that tests for html in a rendered view?